### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by bhushan354**
In This PR I have Implemented following things : 

- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- For every page,  created a <Route>.
- Made sure that navigation links for all 3 routes are displayed on each page.

Thank You ,
Bhushan

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,44 @@
...
+              <Link to="/quote">Quote</Link>
+            </li>
+          </div>
+        </div>
+      </nav>
+      <div className="container">
+        <Routes>
+          <Route path="/home" element={<Home />} />
+          <Route path="/quote" element={<DisplayQuote />} />
+          <Route path="/calculator" element={<Calculator />} />
```
Comment (levy002) [inline src/App.js]: - [x] Kindly make sure that the `Home` component will be displayed the first time we open your and it should act as the homepage of your great application in order to match the project's requirements.

```suggestion
      <nav>
        <div className="navContainer">
          <div>
            <li className="heading">
              Math Magician
            </li>
          </div>

          <div className="linksContainer">
            <li>
              <Link to="/">Home</Link>
            </li>
            <li>
              <Link to="/calculator">Calculator</Link>
            </li>
            <li>
              <Link to="/quote">Quote</Link>
            </li>
          </div>
        </div>
      </nav>
      <div className="container">
        <Routes>
          <Route path="/" element={<Home />} />
          <Route path="/quote" element={<DisplayQuote />} />
          <Route path="/calculator" element={<Calculator />} />
```

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,44 @@
...
+          {/* <DisplayQuote />
+        <Calculator /> */}
```
Comment (levy002) [inline src/App.js]: - [x] Kindly make sure to remove the commented code in order to keep your codebase clean and professional.

File: src/components/Calculator.css [hunk 3]
```diff
@@ -52,14 +53,21 @@ body {
   color: aliceblue;
 }
 
+h3 {
+  color: darkblue;
+  text-align: center;
+  padding-right: 200px;
+  padding-left: 200px;
+}
+
```
Comment (levy002) [inline src/components/Calculator.css]: - [x] Kindly make sure to align correctly the paragraph on the homepage as it looks on the provided wireframe designs.

    ## YOUR DESIGN
   chrome-capture-2023-8-21

    ## PROVIDED DESIGN

    chrome-capture-2023-8-21 (1)

File: src/components/Calculator.css [hunk 4]
```diff
@@ -69,11 +77,25 @@ body {
...
+  text-align: center;
+}
+
+.calciText h2 {
+  color: aliceblue;
+  font-size: 3vw;
+  width: 100%;
+  text-align: center;
+}
+
```
Comment (levy002) [inline src/components/Calculator.css]: - [x] Kindly make sure to align correctly the main title on the homepage as it looks on the provided wireframe designs.

    ## YOUR DESIGN
   chrome-capture-2023-8-21

    ## PROVIDED DESIGN

    chrome-capture-2023-8-21 (1)

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @bhushan354   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 
- Good UI :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @bhushan354👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- Implemented all changes requested by the previous reviewer✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**